### PR TITLE
Bump version to 1.0.0-X19-SNAPSHOT

### PR DESCRIPTION
**Pull request by Krever**
As requested by @jvican

### CODE REVIEW

**Comment by typesafe-tools** [comment]
To validate this pull request against other sbt modules, please visit <A HREF="https://jenkins.scala-sbt.org:8592/job/sbt%20validator/parambuild/?zinc=pull/342/head&deploy_to_bintray=false">this link.</A>

**Comment by jvican** [comment]
@Krever Thank you!
  
@sbt/zinc-team: The reason why we bump up the version is because if we publish the bridges locally, the sbt 1.0.0 build will pick them up. This is an issue when we publish binary incompatible changes, because they fail to link when we try to compile anything else in sbt.
  
That's why it's important that zinc artifacts are always one version ahead of the version that is used in sbt 1.0.

**Review by jvican — COMMENTED** [review]
LGTM

**Review by jvican — DISMISSED** [review]

File: build.sbt [hunk 1]
```diff
@@ -3,7 +3,7 @@ import Dependencies._
 import Scripted._
 // import com.typesafe.tools.mima.core._, ProblemFilters._
 
-def baseVersion = "1.0.0-X16-SNAPSHOT"
+def baseVersion = "1.0.0-X17-SNAPSHOT"
```
Comment (dwijnand) [inline build.sbt]: Should be `1.0.0-X19-SNAPSHOT`

**Review by dwijnand — DISMISSED** [review]

File: build.sbt (hunk 1 above)
Comment (jvican) [reply build.sbt]: Oh wow... Are we that out-of-sync? @Krever Can you update the PR?

**Comment by typesafe-tools** [comment]
To validate this pull request against other sbt modules, please visit <A HREF="https://jenkins.scala-sbt.org:8592/job/sbt%20validator/parambuild/?zinc=pull/342/head&deploy_to_bintray=false">this link.</A>

File: build.sbt (hunk 1 above)
Comment (Krever) [reply build.sbt]: Done

**Review by jvican — APPROVED** [review]
LGTM, again!

**Review by Duhemm — APPROVED** [review]
LGTM, Thanks!

**Comment by typesafe-tools** [comment]
To validate this pull request against other sbt modules, please visit <A HREF="https://jenkins.scala-sbt.org:8592/job/sbt%20validator/parambuild/?zinc=pull/342/head&deploy_to_bintray=false">this link.</A>

**Pull request merged**